packages/react-sdk/src/store/api/documentSnapshotApi.ts

```diff
@@ -68,14 +72,20 @@ export const documentSnapshotApi = baseApi.injectEndpoints({
       },
 
       // do the initial loading
-      async queryFn({ documentId, validator }, { extra }) {
+      async queryFn({ documentId, validator, roomId }, { extra }) {
         const widgetApi = await (extra as ThunkExtraArgument).widgetApi;
 
         try {
+          const roomIdParameter: string | undefined =
```

Comment:
Can you add a comment here, so that we can remember why it is like it is in half a year or so? :D

Reply:
sure, updated with a comment